rest-client.gemspec

```diff
@@ -24,6 +26,7 @@ Gem::Specification.new do |s|
   s.add_dependency("http-cookie", ">= 1.0.2", "< 2.0")
   s.add_dependency("mime-types", ">= 1.16", "< 4.0")
   s.add_dependency("netrc", "~> 0.8")
+  s.add_dependency "ffi", ">= 1.15.5", "<= 1.16.3" if Gem.win_platform?
```

Comment:
This is ok, but I think it will fail if we ever publish a gem vs. installing from branch (like we have to do anyway because we don't control the base of the fork)